R/data-storage-mssqlserver.R

```diff
@@ -0,0 +1,126 @@
+#' Data storage class with MS SQL Server provider
+#'
+#' @description
```

Comment:
Can you add here the additional requirements to use this storage backend? that is, mentioning it needs a configured ODBC driver.

I'm sure that developers who will use this backend will know, but it would be nice to have it documented anyway.

Reply:
Hi @averissimo - thanks for this. I've noted this in the latest commit, at the end of the Description section. I also added a note about the modified schema.